### ISSUE TITLE
Accessibility - Review Incident Screen - TalkBack Reading Form LABELs Incorrectly

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-Android#44]
Starting with the "Category" field until the "Description" field TalkBack is adding "No XXX provided" after reading the content after what looks like a LABEL. The XXX would be the text before the field. If the information is missing, it reads "No XXX provided" twice.

Please see Incident #32 and look at the "Safety & Health Manager" and Garage Section" fields with TalkBack for no data entered which is repeated twice.
